tsfc/kernel_interface/common.py

```diff
@@ -298,10 +299,16 @@ def set_quad_rule(params, cell, integral_type, functions):
     # Check if the integral has a quad degree or quad element attached,
     # otherwise use the estimated polynomial degree attached by compute_form_data
     quad_rule = params.get("quadrature_rule", "default")
+    elements = []
```

Comment:
It seems like mixed coefficients are not split at this point if we have an extruded mesh. Do we want to instead make sure that they are properly split?

Reply:
I can totally believe that that's a bug. Feel free to open an issue or PR about it.